Tools/WinMLRunner/WinMLRunner.vcxproj

```diff
@@ -148,7 +147,6 @@
       <WarningLevel>Level3</WarningLevel>
       <Optimization>Disabled</Optimization>
       <SDLCheck>true</SDLCheck>
-      <ShowIncludes>true</ShowIncludes>
```

Comment:
What is this for?

Reply:
https://docs.microsoft.com/en-us/cpp/build/reference/showincludes-list-include-files?view=vs-2017